docs/plugins/event_listeners.rst

```diff
@@ -1,2 +1,156 @@
 Event listeners
 ===============
+
+Mautic leverages Symfony's EventDispatcher to execute and communicate various actions through Mautic. Plugins can hook into these to extend the capability of Mautic. Refer to the Extending Mautic section of the documentation for some of the ways to do this.
+
+.. code-block:: php
+
+    <?php
+    //plugins\HelloWorldBundle\EventListener\LeadSubscriber
+
+    namespace MauticPlugin\HelloWorldBundle\EventListener;
+
+    use Mautic\LeadBundle\Event as Events;
+    use Mautic\LeadBundle\LeadEvents;
+
+    /**
+     * Class LeadSubscriber
+     *
+     * @package Mautic\LeadBundle\EventListener
+     */
+    class LeadSubscriber extends CommonSubscriber
+    {
+    
+        /**
+         * @return array
+         */
+        static public function getSubscribedEvents()
+        {
+            return array(
+                LeadEvents::LEAD_POST_SAVE     => array('onLeadPostSave', 0),
+                LeadEvents::LEAD_POST_DELETE   => array('onLeadDelete', 0)
+            );
+        }
+    
+        public function onLeadPostSave(LeadEvent $event)
+        {
+            $lead = $event->getLead();
+            
+            // do something
+        }
+    
+        public function onLeadDelete(LeadEvent $event)
+        {
+            $lead = $event->getLead();
+            
+            $deletedId = $lead->deletedId;
+            
+            // do something
+        }
+    }
+    // ...
+
+Subscribers
+-----------
```

Comment:
```suggestion
***********
```

Also note: headings must _always_ be unique. This has a high likelihood of breaking that rule. Could you make it more specific to this particular context?